subprojects/tooling-api/src/integTest/groovy/org/gradle/integtests/tooling/r35/BuildProgressCrossVersionSpec.groovy

```diff
@@ -67,31 +67,18 @@ class BuildProgressCrossVersionSpec extends ToolingApiSpecification {
         then:
         events.assertIsABuild()
 
-        def configureBuild = events.operation("Configure build")
```

Comment:
it's important that the new events appear in the correct location, ie children of the relevant configure project event, so we should test for this.